tests/testthat/test-dummy.R

```diff
@@ -0,0 +1,16 @@
+test_that("dummy works", {
+  pharmaverseadam::adsl |>
+    dummy(adlb = pharmaverseadam::adlb) |>
+    expect_no_error() |>
+    expect_named(c(names(pharmaverseadam::adsl), "ALBBLGRP")) |>
+    with(any(is.null(ALBBLGRP))) |>
+    expect_false()
```

Comment:
[nitpick] Using `with(any(is.null(...)))` can be unclear; consider `expect_false(any(is.na(ALBBLGRP)))` or `expect_true(all(!is.na(ALBBLGRP)))` for clearer intent.
```suggestion
    expect_false(any(is.na(ALBBLGRP)))
```